src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -900,45 +922,44 @@ def load_state(self, model_id: str, evaluation_time: float, epsilon: float = 1e-
 
             BU732106_WORKAROUND = True
             if BU732106_WORKAROUND:
-                # Rather we call a step by step evaluation with a small time-step OR we use the registry outputs
+                # Rather we call a step-by-step evaluation with a small time step OR we use the registry outputs
```

Comment:
Can the "we" be deleted or replaced with "you". We are supposed to use second person (you) and not first (I) or third (we) person.

Reply:
Here this is a comment to describe the workaround this bug. I'd like to keep it like that if that is ok for you?